sdk/openai/openai/review/openai.api.md

```diff
@@ -604,6 +604,13 @@ export interface FunctionName {
     name: string;
 }
 
+// @public
+export interface GenerateSpeechFromTextOptions extends OperationOptions {
+    model?: string;
```

Comment:
We shouldn't have this. Is it still required by the service?

If we need to, may be mock it for now?